Image-Factory/Factory.ps1

```diff
@@ -509,7 +571,18 @@ function RunTheFactory
             Copy-Item "$($ResourceDirectory)\bits" -Destination ($driveLetter + ":\") -Recurse;
             
             # Create first logon script
-            $updateCheckScriptBlock | Out-String | Out-File -FilePath "$($driveLetter):\Bits\Logon.ps1" -Width 4096;
+            if($UseStaticIP) {
+                $staticUpdateCheckScriptBlock = $updateCheckScriptBlock | Out-String
+                $staticUpdateCheckScriptBlock = $staticUpdateCheckScriptBlock.Replace('$UseStaticIP = $false', '$UseStaticIP = $true')
```

Comment:
The "$UseStaticIP = $false" gets changed here if it is enabled.  In the scriptblock it is set to false so that nothing needed to be changed if static IP isn't enabled.